app/Integrations/GitHub/GitHubPlugin.php

```
@@ -0,0 +1,428 @@
+                'value_unit' => 'commits',
+                'event_metadata' => [
+                    'ref' => $data['payload']['ref'],
+                    'before' => $data['payload']['before'],
+                    'after' => $data['payload']['after'],
+                ],
+                'blocks' => $blocks,
+            ]],
+        ];
+    }
+    
+    protected function convertPullRequestEvent(array $data, Integration $integration): array
```

PHPMD (2.15.0)

Avoid unused parameters such as '$integration'. (Unused Code Rules)

(UnusedFormalParameter)